src/modelinstance.cpp

```diff
@@ -712,7 +691,6 @@ void ModelInstance::unloadModelComponents() {
     inferRequestsQueue.reset();
     execNetwork.reset();
     network.reset();
-    engine.reset();
```

Comment:
I'd manually check shape=auto functionality to ensure it works even if unit tests pass.
There is a problem with specific models that cannot be reshaped and in this scenario OVMS tries to reshape back to initial shape. For those models reshaping is glitched up to the point you make "full reload" which OVMS performs. In this case even the core is reinitialized.